README.md

```diff
@@ -29,60 +29,34 @@ The current const_fn requires Rust 1.31 or later.
 
 ## Examples
 
-When using like the following functions to control unstable features:
-
-```toml
-[features]
-const_unstable = []
-```
-
-It can be written as follows:
-
 ```rust
-#![cfg_attr(feature = "const_unstable", feature(const_fn))]
 use const_fn::const_fn;
 
-pub struct Foo<T> {
-    x:T,
+// 1.36 and later compiler (including beta and nightly)
+#[const_fn("1.36")]
+pub const fn version() {
+    /* ... */
 }
 
-impl<T: Iterator> Foo<T> {
-    /// Constructs a new `Foo`.
-    #[const_fn(feature = "const_unstable")]
-    pub const fn new(x: T) -> Self {
-        Self { x }
-    }
+// nightly compiler (including dev build)
+#[const_fn(nightly)]
+pub const fn nightly() {
+    /* ... */
```

Comment:
To use the nightly only unstable feature, it needs to enable the feature at crate-level, and in any case, it needs build-script. So, this may not be very useful in practice, but I'll keep it for now.